plugins/modules/podman_image.py

```diff
@@ -667,7 +668,7 @@ def push_image(self):
         dest_string = dest_format_string.format(transport=transport, name=self.name, dest=dest, image_name=self.image_name,)
 
         # Only append the destination argument if the image name is not a URL
```

Comment:
this comment is not true now, right?
` # Only append the destination argument if the image name is not a URL`